src/clients/cli.rs

```diff
@@ -162,14 +162,16 @@ impl Docker for Cli {
     }
 
     fn rm(&self, id: &str) {
-        Command::new("docker")
+        let output = Command::new("docker")
             .arg("rm")
             .arg("-f")
             .arg("-v") // Also remove volumes
             .arg(id)
             .stdout(Stdio::piped())
-            .spawn()
+            .output()
             .expect("Failed to execute docker command");
+        log::error!("{:?}", output);
+        assert!(output.status.success(), "Failed to remove docker container");
```

Comment:
I don't think the debug log is going to very useful as it contains two `Vec<u8>` :sweat_smile: 

Reading up on the docs of [`spawn`](https://doc.rust-lang.org/std/process/struct.Command.html#method.spawn), it seems like stdin, stdout and stderr are inherited meaning if the process fails and the test is run with `--no-capture` then one would see the stderr output and can investigate the failure. That means we can probably also remove line 170 :)

Reply:
Ok, I've made the changes. LMK if it looks good.